mlflow/metrics/genai/model_utils.py

```diff
@@ -116,6 +115,34 @@ def _call_openai_api(openai_uri, payload):
     except Exception as e:
         raise MlflowException(f"Error response from OpenAI:\n {e}")
 
+    return _convert_from_openai_response(resp)
+
+
+def _call_gateway_api(gateway_uri, payload):
+    from mlflow.gateway import query
+
+    return query(gateway_uri, payload)
+
+
+def _call_deployments_api(deployment_uri, payload):
+    from mlflow.deployments import get_deploy_client, get_deployments_target
+
+    target = get_deployments_target()
+    client = get_deploy_client(target)
+
+    payload = _convert_to_openai_request(payload)
+    resp = client.predict(endpoint=deployment_uri, inputs=payload)
+    return _convert_from_openai_response(resp)
+
+
+def _convert_to_openai_request(payload):
+    payload = {{"candidate_count": "n"}.get(k, k): v for k, v in payload.items()}
+    payload["temperature"] = 2 * payload["temperature"]
```

Comment:
No need to scale temperature anymore. Endpoints use the 0-2 range from OpenAI

Reply:
this is needed for now because we are currently translating gateway -> openai format. so we are going from 0-1 to the 0-2 that both deployments and openai use. 

once we update this code to be "openai format based" rather than "gateway format based" we update this with the inverse direction so that we can translate openai -> gateway. i am assuming this is needed for backward compat, but lmk if this will not needed.

Reply:
ah got it - thanks!